src/lib/Commands/InitCommand.js

```diff
@@ -119,10 +143,10 @@ class InitCommand extends AbstractCommand {
     for (const modelId in this._models) {
       for (const deviceId of  this._models[modelId].devices) {
         table.push([
-          deviceId,
-          this._devices[deviceId].name,
-          this._models[modelId].id,
-          this._models[modelId].name,
+          deviceId || "",
```

Comment:
Yep I remember it looks familiar. So I was wondering where it came from ) Thanks for reminding.

Please leave these changes, just alight them so that it looks better please:

```javascript
        table.push([
          deviceId                     || "",
          this._devices[deviceId].name || "",
          this._models[modelId].id     || "",
          this._models[modelId].name   || "",
          this._devices[deviceId].powerstate === 'online'
            ? c.green(this._devices[deviceId].powerstate)
            : c.red(this._devices[deviceId].powerstate)
        ]);
```